m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -475,34 +589,38 @@ PRINT TITLE=""
  CALL INITIALIZEDATA
  CALL SETMEMOMODE
  CALL READCONFIGFILESETTINGS
- IF ERRORCODE=0 THEN
+ IF BNOERRORCODE=0 THEN
+  CALL PARAMPOSTPROCESS
+
+ IF BNOERRORCODE=0 THEN
   CALL CHECKFORPA3
 
-[*If system is not in memo mode, continue
-*]
- IF @RGSTATE=STATEPRELOADDATA THEN
+ IF BNOERRORCODE=0 THEN
   DO
-   CALL READPREFSETTINGS
-
-   IF PARAMENFORCELIMITS=TRUE AND
-    ((PARAMINSTCOUNTLIMIT=0 AND MBRCOUNTLIMIT=0) OR
-     (PARAMINSTAMOUNTLIMIT=$0.00 AND MBRAMOUNTLIMIT=$0.00) OR
-      PARAMINSTINDIVLIMIT=$0.00 AND MBRINDIVIDUALLIMIT=$0.00) THEN
+   IF @RGSTATE=STATEPRELOADDATA THEN
     DO
-     NEWLINE
-     CALL JSONOPEN
-     CALL PRINTPROGRAMINFO
-     ERRORCODE=501
-     CALL ERRORHANDLER
-     @RGSTATE=STATEEND
```

Comment:
Hello @rrobison44 , I've noticed you've removed all of the lines that say "@RGSTATE=STATEEND" in BANNO.M2MTRANSFERS.V3.POW. Is that not needed?